src/sentry/static/sentry/app/views/settings/components/forms/formField/formFieldWrapper.jsx

```diff
@@ -6,32 +6,36 @@ import SettingsInputField from '../styled/input';
 import SettingsTextAreaField from '../styled/textarea';
 
 const FormFieldWrapper = styled(({highlighted, inline, ...props}) => <Flex {...props} />)`
-  padding: 15px 0 15px 20px;
+  padding: 0.9em 0 0.9em 1.3em;
   border-bottom: 1px solid ${p => p.theme.borderLight};
   transition: background 0.15s;
 
-  ${p =>
```

Comment:
found this syntax nearly impossible to read because prettier is formatting it like js. I'm open to suggestions if folks don't like this!

Reply:
According to Emotion's slack channel, this is supposed to be fixed/better in the latest version of Prettier.

Reply:
Oops, disregard. Was looking at the old code, and not the new. +1 on the cleaner formatting.

Reply:
ok i'll revert this